### DISCUSSION TITLE
Redis error: WRONGPASS invalid username-password pair or user is disabled.

### DISCUSSION THREAD

**Discussion by hydeparkk** [docmost/docmost#366] [Help / Questions]
When using redis connection string with username and password, like this
`REDIS_URL=redis://username:password@host:port`

following error is logged:
`ReplyError: WRONGPASS invalid username-password pair or user is disabled.
    at parseError (/app/node_modules/.pnpm/redis-parser@3.0.0/node_modules/redis-parser/lib/parser.js:179:12)
    at parseType (/app/node_modules/.pnpm/redis-parser@3.0.0/node_modules/redis-parser/lib/parser.js:302:14) {
  command: { name: 'auth', args: [ 'redacted' ] }
}`

and application does not work at all.

**Comment by Philipinho** [comment]
Is your redis instance configured to require password?

**Reply by hydeparkk** [reply]
Yes, it needs username and password as we are using ACL in redis.
Currently I've found a temporary solution for this and use the same password for the default user and user configured for docmost instance, but it isn't the cleanest solution to this problem.
